src/web/pages/nvts/nvtpreference.jsx

```diff
@@ -15,127 +16,116 @@ import TextField from 'web/components/form/textfield';
 import YesNoRadio from 'web/components/form/yesnoradio';
 import Column from 'web/components/layout/column';
 import Divider from 'web/components/layout/divider';
-import Layout from 'web/components/layout/layout';
 import TableData from 'web/components/table/data';
 import TableRow from 'web/components/table/row';
 import PropTypes from 'web/utils/proptypes';
 
-const noop_convert = value => value;
+const StyledTableData = styled(TableData)`
+  overflow-wrap: break-word;
+  white-space: normal;
+  word-break: break-word;
+`;
 
-class NvtPreference extends React.Component {
-  constructor(...args) {
-    super(...args);
+const noop_convert = value => value;
```

Comment:
Would you mind converting noop_convert into camel case too in your PR? i think we should convert old stuff to camel case wherever we touch code
```suggestion
const noopConvert = value => value;
```